src/tests/data/raidsData.ts

```diff
@@ -0,0 +1,328 @@
+const fetchedCollectionData = {
+  state: 'success',
+  data: [
```

Comment:
Please reduce the size of the CollectionData - it is needlessly large for what needs to be tested.